chef/cookbooks/neutron/templates/default/neutron.conf.erb

```diff
@@ -849,6 +849,13 @@ pool_timeout = <%= @sql_pool_timeout %>
 # PEM encoded client certificate cert file
 # certfile =
 
+<% if node.roles.include?("neutron-server") -%>
```

Comment:
I didn't find any documentation, but the only files that seem to be using the nova notifier code are neutron/api/v2/base.py and neutron/db/db_base_plugin_v2.py, which, to my knowledge, are specific to the neutron server.